examples/using-mongodb/src/html.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
```

Comment:
Generally you don't need a custom html.js. Just use https://www.gatsbyjs.org/packages/gatsby-plugin-react-helmet and then the react-helmet stuff will get added automatically to the `<head>`